src/marqo/tensor_search/api.py

```diff
@@ -191,6 +193,29 @@ def memory():
     return memory_profiler.get_memory_profile()
 
 
+@app.post('/validate/{index_name}')
```

Comment:
We want to have an endpoint as `/validate/index` if `index_name` is not necessary, so we can have `/validate/model` in the future

Reply:
index name is not needed, I changed it to this I thought that what @farshidz asked me earlier

Reply:
actually index_name is needed because `to_marqo_index_request` method needs it

Reply:
@wanliAlex changed the signature to @app.post('/validate/index/{index_name}')